[BUG] MKS Robin Nano v3 Freezes with 2.0.8

### Did you test the latest `bugfix-2.0.x` code?

No, but I will test it now!

### Bug Description

No more serial port :  
No usb connexion
No tft serial port active

### Bug Timeline

_No response_

### Expected behavior

Connexion with PC or Octoprint
Connexion with MKS TFT 28

### Actual behavior

No connexion with PC or Octoprint and no connexion with the screen

### Steps to Reproduce

_No response_

### Version of Marlin Firmware

2.0.8

### Printer model

Artillery Genius

### Electronics

MKS Robin Nano V3

### Add-ons

_No response_

### Your Slicer

_No response_

### Host Software

OctoPrint

### Additional information & file uploads

_No response_

I tested it. When connected, the board did not hang, a COM port appeared on the PC. There is still no connection:
`Connecting...
[ERROR] Can't write to printer (disconnected?) (SerialException): Write timeout`

> I tested it. When connected, the board did not hang, a COM port appeared on the PC. There is still no connection:
> `Connecting... [ERROR] Can't write to printer (disconnected?) (SerialException): Write timeout`

Unplug and plug the cable.

--- a/Marlin/src/HAL/STM32/msc_sd.cpp
+++ b/Marlin/src/HAL/STM32/msc_sd.cpp
@@ -38,7 +38,7 @@ public:
         return &card.media_usbFlashDrive;
       #endif
     #else
-      return diskIODriver();
+      return card.diskIODriver();
     #endif
   }
 

